wpengine/faustjs
# [MERL-488] Docs: Add Example Project Walkthrough

## Description



Adds Example Project Walkthrough documentation.

## Related Issue(s):



## Testing

* Navigate to the `docs/next/guides/project-walkthrough` page
* Check that link to that page exists in the `docs/next/getting-started` page


## Screenshots


<img width="1779" alt="Screenshot 2022-10-20 at 13 11 21" src="https://user-images.githubusercontent.com/328805/196945291-4c35ab42-681e-42b3-b73e-dbad757c71c4.png">

## Documentation Changes



## Dependant PRs
